spec/services/hubee_api_client_spec.rb

```diff
@@ -0,0 +1,146 @@
+RSpec.describe HubEEAPIClient do
+  let(:hubee_api_client) { described_class.new }
+
+  let(:api_host) { Rails.application.credentials.hubee_host }
+  let(:access_token) { 'some_access_token' }
+
+  before do
+    stub_hubee_auth
+  end
+
+  describe '#get_organization' do
+    subject(:get_organization) { hubee_api_client.get_organization(siret, code_commune) }
+
+    let(:siret) { '21920023500014' }
+    let(:code_commune) { '75001' }
+    let(:organization_payload) { build(:hubee_organization_payload) }
+
+    context 'when organization already exists in HubEE' do
+      before do
+        stub_hubee_get_organization(siret, code_commune, organization_payload)
+      end
+
+      it 'calls the stubbed request' do
+        get_organization
+
+        expect(a_request(:get, "#{api_host}/referential/v1/organizations/SI-#{siret}-#{code_commune}")
+                 .with(headers: { 'Authorization' => "Bearer #{access_token}" })).to have_been_requested
+      end
```

Comment:
Je vérifierai que le stub exact a été appelé.

```suggestion
      let!(:stubbed_hubee_get_organization_request) do
        stub_hubee_get_organization(siret, code_commune, organization_payload)
      end

      it 'calls the stubbed request' do
        get_organization

        expect(stubbed_hubee_get_organization_request).to have_been_requested
      end
```

Reply:
C'est valide pour tous en bas btw

Reply:
Ah j'ai pigé en relisant :+1: